src/main/java/org/junit/runners/model/FrameworkMethod.java

```diff
@@ -149,6 +149,11 @@ public boolean isShadowedBy(FrameworkMethod other) {
     }
 
     @Override
+    boolean isBridgeMethod() {
```

Comment:
@kcooney Is the assumption correct, that every bridge method is also synthetic? Couldn't find a definitive answer to that question in the language spec or via googling, yet.